src/Modules/MotdModule/Interfaces/IMotdSettings.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Config.Net;
+using EvoSC.Modules.Attributes;
+
+namespace EvoSC.Modules.Official.MotdModule.Interfaces;
+
+[Settings]
+public interface IMotdSettings
+{
+    [Option(DefaultValue = "https://directus.evoesports.de/items/motd?filter[server][_eq]=testserver")]
```

Comment:
Since this is an internal module for a public application that anyone can use, we should not have specific default values that point to our own services if not really needed. I believe we should also provide some kind of API description for how the users can set this up themselves. This also means that the URL fetch should probably be disabled by default.